drivers/SmartThings/matter-switch/fingerprints.yml

```
@@ -2080,6 +2080,22 @@ matterManufacturer:
     vendorId: 0x139C
     productId: 0xEEE2
     deviceProfileName: switch-binary
+  - id: "5020/53249"
+    deviceLabel: Zemismart downlight
+    vendorId: 0x139C
+    productId: 0xD001
+    deviceProfileName: light-color-level
+  - id: "5020/53250"
+    deviceLabel: Zemismart bulb
```

```suggestion
    deviceLabel: Zemismart Bulb
```
Should this be capitalized? That seems to be the format followed throughout